app.js

```diff
@@ -12,16 +12,23 @@ const annotateArticles = require("./lib/annotate.js")
 
 // Constants
 const LOCAL_PORT = 3000 // If this is being run locally then do it on this port
+var categoryList = ['all', 'business', 'entertainment', 'general', 'health', 'science', 'sports', 'technology']
```

Comment:
Avoid using var as much as possible because var does not have block scope, only function scope and this can result in scope confusion
```suggestion
let categoryList = ['all', 'business', 'entertainment', 'general', 'health', 'science', 'sports', 'technology']
```